notebooks/community/matching_engine/matching_engine_for_indexing.ipynb

```diff
@@ -261,7 +261,7 @@
         "\n",
```

Comment:
<p>Why preview version?</p><p> </p><p>Vertex SDK -&gt; Vertex AI SDK</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1465/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>Notebook was a bit outdated. Fixed to use the latest GA version of Vertex AI SDK</p>